xml/System.Net.Http/HttpClientHandler.xml

```diff
@@ -396,8 +396,8 @@ After NuGet package v4.3.2, the default value of <xref:System.Net.DecompressionM
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets or sets the maximum number of concurrent connections allowed by an <see cref="T:System.Net.Http.HttpClient" /> object.</summary>
```

Comment:
This property is about connections per server endpoint when making requests using HttpClient.  So, the limit would control how many concurrent connections are made when accessing, for example, http://www.bing.com.  But if accessing http://www.microsoft.com, then that endpoint has its own limit.  So, if the property were 256, then 256 concurrent connections can be made to bing.com and 256 concurrent connections can be made to microsoft.com at the same time.